examples/llm/src/models/param_init_fns.py

```diff
@@ -134,6 +140,16 @@ def generic_param_init_fn_(module, cfg, init_fn_):
         if module.bias is not None:
             torch.nn.init.zeros_(module.bias)
 
+    elif DROPOUT_ADD_LAYERNORM_INSTALLED and isinstance(module,
+                                                        DropoutAddLayerNorm):
+        if cfg.get('verbose', 0) > 1:
+            warnings.warn(
+                f'LayerNorm gamma weights are set to 1.  If the layer has a bias it is initialized to 0.'
+            )
+        torch.nn.init.ones_(module.weight)
+        if module.bias is not None:
+            torch.nn.init.zeros_(module.bias)
```

Comment:
why not combine this with the `elif isinstance(module, nn.LayerNorm): ` section?
its just a longer if statement.